src/autopas/AutoPas.h

```diff
@@ -465,7 +467,7 @@ class AutoPas {
 
   /**
    * Set the selector configuration strategy.
```

Comment:
Needs to be more descriptive. Something like this:
```suggestion
   * Set the strategy of how to select a performance value for a piece of evidence from multiple time measurements (=samples).
```